resume/src/styles.css

```diff
@@ -0,0 +1,4 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Note for documentation: do you have to add this to the styles.css by yourself or does Tailwind do that on setup?

Do you think we should mention how to set up an empty Tailwind project or maybe link to a Tutorial for future reference?